samples/sqltransport/multi-instance/SqlServer_3/Sender/Program.cs

```diff
@@ -68,4 +69,25 @@ await endpoint.Send("Samples.SqlServer.MultiInstanceReceiver", order)
         Console.WriteLine($"ClientOrder message sent with ID {order.OrderId}");
     }
 
+    #region SenderConnectionProvider
+    static class ConnectionProvider
+    {
+        const string DefaultConnectionString = @"Data Source=.\SqlExpress;Database=SenderCatalog;Integrated Security=True";
+        const string ReceiverConnectionString = @"Data Source=.\SqlExpress;Database=ReceiverCatalog;Integrated Security=True";
+
+        public static async Task<SqlConnection> GetConnection(string transportAddress)
+        {
+            var connectionString = transportAddress.StartsWith("Samples.SqlServer.MultiInstanceReceiver")
```

Comment:
This is the reverse to Sender. The receiver returns default connection string, except when message is destined to Receiver.

In general this the approach that we want to support and that's why the sample was changed. The approach is:
  * By default return a (no surprise  here ;)) default connection string
  * For each **remote** destination have an explicit match and return non default connection string.